omnia/test/gasprice.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+test_path=$(cd "${BASH_SOURCE[0]%/*}"; pwd)
+root_path=$(cd "$test_path/.."; pwd)
+lib_path="$root_path/lib"
+
+. "$lib_path/log.sh"
+. "$lib_path/gasprice.sh"
+
+. "$root_path/tap.sh" 2>/dev/null || . "$root_path/../tests/lib/tap.sh"
+
+getGasPriceFromNode () {
+  echo "1"
+}
+export -f getGasPriceFromNode
+
+assert "multiplyGasPrice should return multiplied values" match "2" < <(capture multiplyGasPrice 1 2)
```

Comment:
This test will NOT fail when you use `multiplyGasPrice 1 20`
Please add `^` and `$`.